eng/common/template-guidance.md

```diff
@@ -57,7 +57,7 @@ extends:
 
 Note: Multiple outputs are ONLY applicable to 1ES PT publishing (only usable when referencing `templates-official`).
 
-## Development notes
+# Development notes
```

Comment:
Fix `Multiple top-level headings in the same document [Context: "# Development notes"]`. PR opened upstream - https://github.com/dotnet/arcade/pull/15082 .
```suggestion
## Development notes
```